mava/components/training/losses.py

```diff
@@ -28,6 +29,91 @@
 from mava.core_jax import SystemTrainer
 
 
+class DefaultValueLossFunction(Component):
```

Comment:
Since this is setting up a SE loss, I think we should rename this to `SquaredErrorLoss`? We can also drop the "Function" in the class names to make them shorter.

Reply:
+1